modules/alb-logs-to-s3-to-cloudwatch/logs-to-cloudwatch.tf

```diff
@@ -4,9 +4,10 @@ data "aws_s3_bucket" "selected" {
 }
 
 resource "aws_cloudwatch_log_group" "log" {
```

Comment:
I suggest deleting this module from the repo, cause it's already exists in eks module.